plugins/woocommerce/i18n/locale-info.php

```diff
@@ -3,19 +3,37 @@
  * Locales information
  *
  * @package WooCommerce\i18n
- * @version 5.7.0
+ * @version 8.1.0
```

Comment:
See my comment on `plugins/woocommerce/i18n/currency-info.php` -- this should probably be:

```suggestion
 * @version 8.2.0
```

Reply:
Addressed in https://github.com/woocommerce/woocommerce/pull/40101/commits/3de8ed8c55df591024db9f25148e908d87376228.